front/src/modules/SimulationReportSheet/SimulationTable.tsx

```diff
@@ -0,0 +1,144 @@
+import { Table, TR, TH, TD } from '@ag-media/react-pdf-table';
+import { Link, Text, View } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import styles from './styles/SimulationReportStyleSheet';
+import type { SimulationTableScenarioProps, SimulationTableStdcmProps } from './types';
+import formatSimulationTable from './utils/formatSimulationTable';
+
+type SimulationTableProps =
+  | (SimulationTableStdcmProps & { mode: 'stdcm' })
+  | (SimulationTableScenarioProps & { mode: 'operationalStudies' });
+
+const SimulationTable = (props: SimulationTableProps) => {
+  const { t } = useTranslation('stdcm');
+  const { mode } = props;
+  const isStdcm = mode === 'stdcm';
+  const rows = formatSimulationTable({ ...props });
+
+  const path = isStdcm
+    ? (props as SimulationTableStdcmProps & { mode: 'stdcm' }).stdcmData.path
+    : (props as SimulationTableScenarioProps & { mode: 'operationalStudies' }).path;
+
+  const lengthInKm = Math.round(path.length / 1000000);
+
+  return (
+    <View style={styles.simulation.simulation}>
+      <View style={styles.simulation.simulationContainer}>
+        <Text style={styles.simulation.simulationUppercase}>{t('reportSheet.simulation')}</Text>
+        {mode === 'operationalStudies' && (
+          <Link href="#simulationMap" src="#simulationMap" style={styles.simulation.viewSimulation}>
+            {t('reportSheet.viewSimulation')}
+          </Link>
+        )}
+        <Text style={styles.simulation.simulationLength}>{`${lengthInKm} km`}</Text>
```

Comment:
- use mmToKm utils
- I think the code is simple here, no need for a intermediate const
```suggestion
        <Text style={styles.simulation.simulationLength}>{`${Math.round(mmToKm(path.length))} km`}</Text>
```

Reply:
done